content/en/docs/contribute/advanced.md

```diff
@@ -2,14 +2,14 @@
 title: Advanced contributing
 slug: advanced
 content_template: templates/concept
-weight: 30
+weight: 98
 ---
 
 {{% capture overview %}}
 
-This page assumes that you've read and mastered the
-[Start contributing](/docs/contribute/start/) and
-[Intermediate contributing](/docs/contribute/intermediate/) topics and are ready
+This page assumes that you've already
+[contributed to new existing content](/docs/contribute/new-content/overview) and
+[learned how to review others' work](/docs/contribute/review/reviewing-prs/), and are ready
```

Comment:
nit: This page assumes that you understand how to contribute new content, review pull requests, and are ...